starknet_py/net/client_errors.py

```diff
@@ -8,9 +8,15 @@ class ClientError(Exception):
 
     def __init__(self, message: str, code: Optional[str] = None):
         self.code = code
-        self.message = f"Client failed{f' with code {code}' if code is not None else ''}: {message}"
+        self.message = message
         super().__init__(self.message)
 
+    def __str__(self):
+        return (
+            f"Client failed{f' with code {self.code}' if self.code is not None else ''}"
+            f": {self.message}"
+        )
+
```

Comment:
The interesting change I would like to keep here is the fact that `.message` only includes the `message` argument, and not a modified version appending "Client failed...". It would ease using the `message` attribute outside the project (else I would need to "parse" the message, and try to get back the original value, which is not really future-proof nor neat).